### PR TITLE
Rn test caseupdate django upgrade

### PR DESCRIPTION
**Pull request by Rohit25negi**
Fixing test cases for the icds_reports app.

~~Not open for review right now.~~
This is now ready for review

### CODE REVIEW

**Comment by Rohit25negi** [comment]
This is now open for review

File: custom/icds_reports/tests/agg_tests/__init__.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 OUTPUT_PATH = os.path.join(os.path.dirname(__file__), 'outputs')
 
 
-@timelimit(480)
+@timelimit(48000000)
```
Comment (millerdev) [inline custom/icds_reports/tests/agg_tests/__init__.py:30]: This seems a bit excessive. Maybe we should double it to 960? 480 is already about twice as long as it takes on travis. Did something change to make this much longer?

**Review by snopoke — APPROVED** [review]
How will this change in the data manifest in the reports on the UI?

**Comment by Rohit25negi** [comment]
> How will this change in the data manifest in the reports on the UI?

I think our Report UIs will not have effect of this because we display 'floats' on UI for percentages and for percentages we truncate the percentage to 2 decimals which looks working the same for Decimal as well. So reports should perform similarly the way they were performing. Dev tested those, waiting for QA as well go through it.

**Comment by Rohit25negi** [comment]
> How will this change in the data manifest in the reports on the UI?

I think our Report UIs will not have the effect of this because we display 'floats' on UI for percentages and for percentages we truncate the percentage to 2 decimals which looks working the same for Decimal as well. So reports should perform similarly the way they were performing. The test cases which are changed, are y coordinates for the line chart which should still be the same because for example earlier if it was coming as '0.25' now it will come on UI as '0.250000' which is resolving to the same point on y-axis. So should be good. Dev tested those,  waiting for QA as well go through it.

**Review by millerdev — APPROVED** [review]

**Comment by millerdev** [comment]
The changes in this PR are not backward compatible with Django 1.11, so this can only be merged with/after the Django 2.2 upgrade PR.

**Comment by millerdev** [comment]
Closing in favor of https://github.com/dimagi/commcare-hq/pull/28139

**Pull request closed without merging**